pyproject.toml

```diff
@@ -25,6 +25,9 @@ dev = [
 example = [
     "trl",
 ]
+faiss = [
+    "faiss-gpu-cu12",
```

Comment:
can support CPU-only FAISS in the future (just need to set device to CPU or use the mmap config option)